src/helpers/util.ts

```diff
@@ -106,9 +106,12 @@ export function sortQueriesByDate({
   propose: OracleQueryList;
   settled: OracleQueryList;
 }) {
+  // propose and settled are sorted by the time the query was created
+  // verify is sorted by when the liveness ends, so that the ones that end soonest are easy to find
   return {
     verify: verify.sort(
-      (a, b) => (b.timeMilliseconds || 0) - (a.timeMilliseconds || 0)
+      (a, b) =>
+        (b.livenessEndsMilliseconds || 0) - (a.livenessEndsMilliseconds || 0)
```

Comment:
not sure we are getting the right behavior
![image](https://user-images.githubusercontent.com/4429761/233368236-4b12bfc3-5e21-4c19-a0bd-78740be227ea.png)
we should see the shorter liveness be above the longer one